lib/groonga/client.rb

```diff
@@ -33,6 +33,10 @@ class << self
       #   @param [Hash] options The options.
       #   @option options [:gqtp or :http] :protocol The protocol that is
       #     used by the client.
+      #   @option options [String] :auth_user User ID for HTTP Basic
```

Comment:
How about removing `auth_` prefix? Because we don't have any other use cases for user and password in client.